src/Features/Core/Portable/ExternalAccess/UnitTesting/API/NewUnitTestingIncrementalAnalyzer.cs

```diff
@@ -25,6 +25,7 @@ public Task AnalyzeDocumentAsync(Document document, SyntaxNode bodyOpt, UnitTest
         public Task AnalyzeProjectAsync(Project project, bool semanticsChanged, UnitTestingInvocationReasons reasons, CancellationToken cancellationToken)
             => _implementation.AnalyzeProjectAsync(project, semanticsChanged, reasons, cancellationToken);
 
+#if false // Not used in unit testing crawling
```

Comment:
all the methods that unit testing just returns Task.CompletedTask are not necessary to support.  so each commit ifdefs one out and works backwards, removing anything uneeded.